### DISCUSSION TITLE
Generated output insertion points

### DISCUSSION THREAD

**Discussion by wulfgarpro** [protobuf-c/protobuf-c#614] [Q&A] [answered]
Should I not expect the output of --c_out to include [insertion points](https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#plugins)?
I'd like to add additional include directives to the resultant header file.

I'm new to protobuf and plugins; does the protobuf-c plugin apply its generator to the output of the --cpp_out directive? This output does include the insertion points -- am I able to perform the insertion here before protobuf-c generates its c out?

**Comment by edmonds** [comment, marked as answer]
This is protobuf-c, a C implementation of Protocol Buffers that depends on components from Google's protobuf implementation. It is unrelated to the C++ protobuf code generator.

**Comment by wulfgarpro** [comment]
@edmonds, OK, so, nothing comparable exists in protobuf-c?
